File: Angularjs-Application-feb26th/project2/WebContent/Controller/BlogController.js
/**
 * 
 */
app.controller('BlogController',function($scope,BlogService)
		{
	$scope.addblog=function(blog)
	{
	BlogService.addblog(blog).then(function(response){
		alert("Su")
		
	},function(response)
	{
	
		alert("failre")
		
	})
		}
	
	
})

File: Angularjs-Application-feb26th/bridgeware/src/main/java/com/project2/controller/BlogController.java
package com.project2.controller;

import org.springframework.beans.factory.annotation.Autowired;
import org.springframework.web.bind.annotation.RequestBody;
import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.RequestMethod;
import org.springframework.web.bind.annotation.RestController;

import com.project2.DAO.BlogDAO;
import com.project2.model.Blog;

@RestController
public class BlogController 
{
@Autowired
private BlogDAO blogdao;
@RequestMapping(value="/addblog",method=RequestMethod.POST)
public void addUser(@RequestBody Blog blog)
{
	blogdao.addBlog(blog);
	
	
}

	

}
